go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -710,22 +710,22 @@ func getEndpointAddresses(endpointSlice *discovery.EndpointSlice) []string {
 func (npw *nodePortWatcher) UpdateEndpointSlice(oldEpSlice, newEpSlice *discovery.EndpointSlice) {
 	oldEpAddr := getEndpointAddresses(oldEpSlice)
 	newEpAddr := getEndpointAddresses(newEpSlice)
+	// should we sort the slices before comparing them? maybe not?
```

Comment:
yes, it's not needed

Reply:
Cool! Let me remove that line... I actually didn't mean to push it, my bad! :)